pkgs/tools/networking/network-manager/l2tp.nix

```diff
@@ -5,13 +5,13 @@
 stdenv.mkDerivation rec {
   name    = "${pname}${if withGnome then "-gnome" else ""}-${version}";
   pname   = "NetworkManager-l2tp";
-  version = "1.2.4";
+  version = "1.2.8";
 
   src = fetchFromGitHub {
     owner  = "nm-l2tp";
     repo   = "network-manager-l2tp";
     rev    = "${version}";
-    sha256 = "1mvn0z1vl4j9drl3dsw2dv0pppqvj29d2m07487dzzi8cbxrqj36";
+    sha256 = "ca39414a0282a2f3457df952bc2a5b386c21b4bd21582bbc6c48dadd5fb5dfff";
```

Comment:
Are you sure about the checksum?

```
output path ‘/nix/store/rnz4amy2sjmgmipsf4b0lw3sxkv8fajc-network-manager-l2tp-1.2.8-src’ has r:sha256 hash ‘110157dpamgr7r5kb8aidi0a2ap9z2m52bff94fb4nhxacz69yv8’ when ‘1zyznmgxvnj8djy2nn11pns22v1qbcmbqlprgm2z78l209542ffa’ was expected
```

Reply:
@Mic92 mmm, I'm not sure how to get that since I used `sha256sum 1.2.8.tar.gz` to get that value, but `sha256sum 1.2.4.tar.gz` doesn't return the previous value so I must be doing it wrong.

Reply:
@ilpianista `fetchFromGitHub` derives the checksum from the archive contents, not the archive itself. You could try e..g, `nix-prefetch-url --unpack https://github.com/nm-l2tp/network-manager-l2tp/archive/$version.tar.gz`.